api/DOMTokenList.json

```
@@ -772,10 +772,10 @@
             "description": "<code>force</code> argument",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "25"
```

```suggestion
                "version_added": "24"
```

Per https://github.com/mdn/browser-compat-data/pull/8482#issuecomment-759483886